mypy_django_plugin/config.py

```diff
@@ -80,10 +83,10 @@ def parse_toml_file(self, filepath: Path) -> None:
         except KeyError:
             toml_exit(MISSING_SECTION.format(section="tool.django-stubs"))
 
-        if "django_settings_module" not in config:
+        self.django_settings_module = config.get("django_settings_module") or os.getenv(DJANGO_SETTINGS_ENV_VAR)
```

Comment:
For the sake of backwards compatibility, I elected to default to config instead of the env var